docs/source/input_preparation.rst

```diff
@@ -0,0 +1,115 @@
+Instructions for preparing inputs for the OpenFE public benchmark
+#################################################################
+
+Overview & Aims
+***************
+
+This document provides guidance on preparing the `Schrodinger benchmark files <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0/fep_benchmark_inputs/structure_inputs>`_ for use in the OpenFE 2024 public dataset industry benchmark.
+
+Whilst each partner organization will be responsible for preparing inputs using their own tooling, this guidance aims to provide a set of rules to standardize the process and ensure that scientifically equivalent simulation inputs are generated.
+
+This document will be continually updated based on feedback from benchmark partners. Please reach out should you have any questions or require additional information whilst preparing your inputs.
+
+Input preparation instructions
+******************************
+
+Input data source
+=================
+
+All input data are sourced from the `v2.0 release of the Schrodinger free energy benchmark <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0>`_. Specifically, input PDB, SDF and edges CSV files can be found under the relevant `structure input <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0/fep_benchmark_inputs/structure_inputs>`_ sub-folder.
+
+For your convenience, a snapshot of these inputs have been placed under the OpenFE Public Benchmark repository. We recommend that you clone this repository and use those input files.
+
+Extracting cofactors
+====================
+
+In its benchmark inputs, Schrodinger places cofactor molecules within the PDB file. Currently OpenFE cannot parse such inputs as small molecules are expected as separate inputs.
+
+Should your input PDB have cofactors, please extract these from the PDB and store them within an SDF file named “cofactors.sdf”. You may be required to manually assign and/or correct bond orders such that they represent the expected state of the molecules and the SDF files are readable by RDKit.
+
+.. note::
+   Waters and ions do not need to be extracted from the PDB file.
+
+Capping proteins
+================
+
+Capping protein depends on how the protein’s termini are handled in the Schrodinger provided PDB files.
+
+If the termini has either a neutral proton caps, e.g. NH2 or C(=O)H, or ACE/NME caps:
+Assign ACE and/or NME caps to the termini
```

Comment:
Worth checking how this renders, might need to do subheadings here or some bolding.